.cirrus.yml

```diff
@@ -33,7 +33,7 @@ env:
     UBUNTU_NAME: "ubuntu-2204"
 
     # Image identifiers
-    IMAGE_SUFFIX: "c5178639502278656"
+    IMAGE_SUFFIX: "c5069932136759296"
```

Comment:
ISTR a conversation about, every time we change the `c` number, adding a link to the `automation_images` PR that produced it. Am I misremembering?

Reply:
Oh right, but it's the same PR :confused:  I can still mention it though, np.  Thanks for suggesting.